src/rules/floats.ts

```diff
@@ -0,0 +1,9 @@
+import { RuleFn } from "./RuleConfig";
+import { makeRules } from "../utils";
+
+export const floatRules: RuleFn = () =>
```

Comment:
Maybe drop in the link `// https://github.com/tachyons-css/tachyons/blob/master/src/_floats.css`

Reply:
Will do!